app/src/main/java/com/smelnykov/homework_1/Constants/Constants.kt

```diff
@@ -10,8 +10,18 @@ object Constants {
      * */
     const val EMAIL_AUTH_KEY = "email"
 
+    /**
+     * Constant for password key
+     * */
+    const val PASSWORD_AUTH_KEY = "password"
```

Comment:
it's arguable, but often people don't like all constants stored in one file and prefer it divided into separate files\classes, or even companion objects of respective classes, e.g. this constant would probably be in the auth activity companion

Reply:
Thank you! Made changes. Commit 9e0bcee7